community-operators/rook/rook.v0.1.0.clusterserviceversion.yaml

```
@@ -0,0 +1,273 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "ceph.rook.io/v1",
+          "kind": "CephCluster",
+          "metadata": {
+            "name": "my-rook-ceph",
+            "namespace": "rook-ceph"
+          },
+          "spec": {
+            "cephVersion": {
+              "image": "ceph/daemon-base:latest-master"
+            },
+            "dataDirHostPath": "/var/lib/rook",
+            "mon": {
+              "count": "3"
+            },
+            "dashboard": {
+              "enabled": true
+            },
+            "network": {
+              "hostNetwork": false
+            },
+            "rbdMirroring": {
+              "workers": "0"
+            },
+            "storage": {
+              "useAllNodes": true,
+              "useAllDevices": true
+            },
+          }
+        },
+      ]
+    tectonic-visibility: ocs
+  name: olm.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Represents a Ceph cluster.
+      displayName: Ceph Cluster
+      kind: CephCluster
+      name: cephclusters.ceph.rook.io
+      version: v1
+    - description: Represents a Ceph Block Pool.
+      displayName: Ceph Block Pool
+      kind: CephBlockPool
+      name: cephblockpools.ceph.rook.io
+      version: v1
+    - description: Represents a Ceph Filesystem.
+      displayName: Ceph Filesystem
+      kind: CephFilesystem
+      name: cephfilesystems.ceph.rook.io
+      version: v1
+    - description: Represents a Ceph NFS interface.
+      displayName: Ceph NFS
+      kind: CephNFS
+      name: cephnfses.ceph.rook.io
+      version: v1
+    - description: Represents a Ceph Object Store.
+      displayName: Ceph Object Store
+      kind: CephObjectStore
+      name: cephobjectstores.ceph.rook.io
+      version: v1
+    - description: Represents a Ceph Object Store User.
+      displayName: Ceph Object Store User
+      kind: CephObjectStoreUser
+      name: cephobjectstoreusers.ceph.rook.io
+      version: v1
+    - description: Represents a Ceph volume.
+      displayName: Ceph Volume
+      kind: Volume
+      name: volumes.rook.io
+      version: v1alpha2
+  description: |
+    Rook Storage Orchestration for Kubernetes
+
+    Rook runs as a cloud-native service for optimal integration with applications in need of storage, and handles the heavy-lifting behind the scenes such as provisioning and management.      ## Before Your Start
```

There is invalid markdown here:

```suggestion
    Rook runs as a cloud-native service for optimal integration with applications in need of storage, and handles the heavy-lifting behind the scenes such as provisioning and management.
    ## Before Your Start
```

---

This section does not cover any pre-requisites as one expect in a `Before you Start` section. It would be great to get a least a hint at some of the pre-reqs outlined here: https://rook.io/docs/rook/v1.0/k8s-pre-reqs.html

Also, consider using `minKubeVersion` to denote your minimum kube version.